### PR TITLE
CMOS-134: Downgrade VirtualBox in cron runs as a workaround for networking issues

### PR DESCRIPTION
**Pull request by markspolakovs**
Seems like VirtualBox 6.1.28 has serious issues with macOS networking that has been causing failed build runs. Downgrade it to 6.1.26 until we figure out a long-term solution.

The way I'm doing the downgrade is horrible, but without `brew switch` there's not many better options. Tracking task for long-term alternatives is https://issues.couchbase.com/browse/CMOS-135.

### CODE REVIEW

File: tools/licence-lint.sh [hunk 1]
```diff
@@ -21,16 +21,16 @@ exitCode=0
 while IFS= read -r -d '' SOURCE
 do
     if head "${SOURCE}" | grep -q '^// Code generated .* DO NOT EDIT.$'; then
-        echo "Skipping licence check for auto-generated file .${SOURCE##$SCRIPT_DIR/..}"
+        echo "Skipping licence check for auto-generated file .${SOURCE##"$SCRIPT_DIR"}"
```
Comment (patrick-stephens) [inline tools/licence-lint.sh:24]: Auto-generated is only under /tools?

File: .github/workflows/cron-build.yml [hunk 2]
```diff
@@ -99,6 +99,13 @@ jobs:
               with:
                 ref: ${{ github.event.inputs.ref || 'main' }}
 
+            - name: Downgrade VirtualBox as a workaround for networking issues
+              # TODO: this is horrible. Remove this when we have a long-term solution.
```
Comment (patrick-stephens) [inline .github/workflows/cron-build.yml:103]: Need to link out a CMOS really, I think we have one somewhere. Really it is a Virtualbox or CB vagrant issue and not ours.

File: tools/licence-lint.sh (hunk 1 above)
Comment (markspolakovs) [reply tools/licence-lint.sh:24]: Copy-paste error. Will fix.

File: .github/workflows/cron-build.yml (hunk 2 above)
Comment (markspolakovs) [reply .github/workflows/cron-build.yml:103]: https://issues.couchbase.com/browse/CMOS-135 - will add to the commit

**Review by patrick-stephens — APPROVED** [review]

**Pull request merged**